docs/tsconfig.json

```diff
@@ -3,6 +3,7 @@
   "extends": "@tsconfig/docusaurus/tsconfig.json",
```

Comment:
```suggestion
  "extends": "@docusaurus/tsconfig",
```

You will need to install this: `yarn add --dev `@docusaurus/tsconfig`

Reply:
See https://github.com/facebook/docusaurus/pull/9050

Reply:
Hmmm, the lock file looks quite different now. Could that be a problem?

Reply:
That is expected, should be an issue